spec/models/comment_spec.rb

```diff
@@ -248,6 +248,14 @@
   end
 
   context "when callbacks are triggered after create" do
+    it "enqueue a worker to create id code" do
```

Comment:
Can we change this description to be `creates an id code`?

Reply:
Done :D